src/Utils/utils.php

```diff
@@ -123,7 +123,7 @@ function get_int_query_var( $var ): int {
 	 *
 	 * @var int
 	 */
-	return get_query_var( $var );
+	return get_query_var( $var, 0 );
```

Comment:
The function was supposed to return an int, but the `get_query_var()` function returns a string by default, which would result in a fatal.